package.json

```diff
@@ -10,6 +10,7 @@
     "@decentralized-identity/did-auth-jose": "0.1.5",
     "@decentralized-identity/did-common-typescript": "0.1.2",
     "base64url": "3.0.1",
+    "bitcore-lib": "^8.3.0",
```

Comment:
Disucssion: do you think it is good to lock version here?

Reply:
I just did `npm install ...` so this was not an intentional decision. In my opinion, locking versions should be left to the `package-lock.json`, but that has a tendency to clash in git commits and just be overwritten, so if you'd like the redundancy, sure?